### DISCUSSION TITLE
Megane R.S cluster / CF_RS values

### DISCUSSION THREAD

**Discussion by danikkm** [cedricp/ddt4all#1157] [Q&A] [answered]
Can someone provide values from the cluster/CF_RS section? Perhaps @fortizen can help?

![image](https://github.com/cedricp/ddt4all/assets/40919040/ced3ae9e-f22a-429d-94c9-ff20f12269bc)

**Comment by fortizen** [comment]
> Can someone provide values from the cluster/CF_RS section? Perhaps @fortizen can help?
> 
> ![image](https://github.com/cedricp/ddt4all/assets/40919040/ced3ae9e-f22a-429d-94c9-ff20f12269bc)

I can't right now, in  4 hours i can as my brother has the car.

**Reply by danikkm** [reply]
Much appreciated

**Reply by danikkm** [reply]
@fortizen any updates?

**Reply by fortizen** [reply]
Shit i forgot, Sorry. Remind me if you can in 2 hours (i will try to keep it in mind)

**Reply by danikkm** [reply]
@fortizen reminder

**Reply by fortizen** [reply]
> @fortizen reminder 

Thanks, On it.

**Reply by fortizen** [reply, marked as answer]
![Screenshot_20240711_223721_Ecu Tweaker.jpg](https://github.com/cedricp/ddt4all/assets/81877833/dff21aa8-f7a6-4c8c-a17b-152f87e6b242)

Mine is a 2020 Trophy, if you need that for the Normal R.S. you should change the hp to 280 and the nm to 390.

**Reply by danikkm** [reply]
Huge thanks!

*I actually installed the Megane R.S. cluster on my Talisman 1.8 TCe and wrongfully reseted all setting to the default.

**Comment by danikkm** [comment]
@fortizen, Hello again! Can you shared the configuration string for the RS monitor, if you have it installed. Thanks!

**Reply by fortizen** [reply]
I'm not sure what do you mean by that. Do you mean the file that is put on the USB Drive for the R-Link to Install it or something else?

**Reply by danikkm** [reply]
Configuration file. The long string (see image bellow)
![IMG_0124](https://github.com/user-attachments/assets/66161814-d6a1-4cea-88a2-164b2a91855f)

**Reply by danikkm** [reply]
Example: 

`00000000000a0bb8010a000508ca010a2ee02ee0659001076e6eaa010a282878010a2ee02ee032c8010a2ee02ee03390010a6e6ea0010a0000190a0a1f402ee03e80010a5f5f91010a6464b4010a00005a010a00008c010a2ee032c884d0010000000000000000000000000000010100010100010180808080757575726b6a686561605e615a58546154514b614f4b44614b463e614a443e614b443e614d443e614d443e618080808083838383858584818887877f8c8b897f908f8c7f97958f7f9e9b8f7fa8a28f7fb3a28f7fbaa28f7fbaa28f7f14506e01320028058223000000000000000000000000006464646400000000000064646464a11e32468ea2aaaaa5a2a09e9a96989a9b96918a82736f6a65635f56001e324bacbec8c9c8c6c8c3b9b4b3b5b6b2aca59b8e6f6a65635f560069019064057864004e2064645f6401903c03e8646464646464640041006700a000e6012c01786464325a6e020796965a829600014502640fdc1c87b402000000000100015f880f010200030414100a0300284b6e96be000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 `

**Reply by danikkm** [reply]
@fortizen, reminder

**Reply by fortizen** [reply]
00000003000a0fa0010a00050bb8010a2ee02ee02f1c01006e6eaa010a282878010a2ee02ee032c8010a2ee02ee03390010a6e6ea0010a00001d0a0a1f402ee03e80010a5f5f91010a6464b4010000008c010000008c010a2ee032c884d0010000000000000000000000000000000000010100010080808080757575726b6a686561605e615a58546154514b614f4b44614b463e614a443e614b443e614d443e614d443e618080808083838383858584818887877f8c8b897f908f8c7f97958f7f9e9b8f7fa8a28f7fb3a28f7fbaa28f7fbaa28f7f14506e0132002805962300000000000000000000000000646464640000000000006464646400505f6e85a0aaaaaab9c8c8c8c8c8c8c6c4bfbbb5b0a69d93918d8989505f6e85a0aaaaaab9c8c8c8c8c8c8c6c4bfbbb5b0a69d93918d898969019064057864004e2064645f6401903c03e864646464646464004b007700a800e6012201606464325a6e020696965a829600014502640fdc1c87b402000000000100015f880f010200030414100a0300284b6e96be000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

**Reply by danikkm** [reply]
Thanks!

**Reply by fortizen** [reply]
No Problem Bro🩵, May i ask what are you working on?

**Reply by danikkm** [reply]
I copied this value from another guy with Espace V with m5t 1.8 the engine. Since I chip-tuned my 1.8 Talisman, I want more information, higher torque values, and other graphs. Knowing that you have Megane RS, you helped me achieve that.
